reference/5.1/Microsoft.PowerShell.Core/About/about_Automatic_Variables.md

```diff
@@ -332,31 +305,36 @@ Contains the name of the user interface (UI) culture that is currently in use in
 
 
 ### $PSVERSIONTABLE
-Contains a read-only hash table that displays details about the version of  Windows PowerShell that is running in the current session. The table includes the following items:
-
+Contains a read-only hash table that displays details about the version of Windows PowerShell that is running in the current session. The table contains the following properties
 
 ```
-CLRVersion:                 The version of the common language runtime (CLR)
+CLRVersion                 The version of the common language runtime (CLR)
 
-BuildVersion:               The build number of the current version
+BuildVersion               The build number of the current version
 
-PSVersion:                  The Windows PowerShell version number
+PSVersion                  The Windows PowerShell version number
 
-WSManStackVersion:          The version number of the WS-Management stack
+WSManStackVersion          The version number of the WS-Management stack
 
-PSCompatibleVersions:       Versions of Windows PowerShell that are  
+PSCompatibleVersions       Versions of Windows PowerShell that are  
                               compatible with the current version
 
-SerializationVersion        The version of the serialization method
+SerializationVersion       The version of the serialization method
 
-PSRemotingProtocolVersion   The version of the Windows PowerShell remote  
+PSRemotingProtocolVersion  The version of the Windows PowerShell remote  
                               management protocol
 ```
 
+In PowerShell 5.1, a new property was added to this automatic variable:
+
+```
+PSEdition                  The edition of PowerShell (Desktop for Windows server and clients)
```

Comment:
This should also include further detail to include and distinguish that there is the possibility that this property can also display Core on Nano Server or on machines that currently have any of the PowerShell v6 alpha's installed.